scalafix/tests/src/test/scala/fix/RuleSuite.scala

```diff
@@ -5,10 +5,8 @@ import scalafix.testkit._
 
 class RuleSuite extends AbstractSemanticRuleSuite with FunSuiteLike {
   val (passing, failing) = testsToRun.partition(!_.path.testName.contains("_fails"))
-
-  passing.foreach(runOn)
-//  failing.foreach(runOn)
-
+//  passing.foreach(runOn)
+  runOnSpecificTest("Basic")
```

Comment:
what the reason for not running on all the tests?

Reply:
I should ^^ there are passing .